### DISCUSSION TITLE
RJ9 to DS18b20 Pinning

### DISCUSSION THREAD

**Discussion by Danieldee1183** [BernhardSchlegel/BierBot-Bricks#36] [General]
Can someone tell me which of the three wires of the sensor belong to which port of the connector? Background: I solder my own from a sensor to a rj9 connector

**Comment by BernhardSchlegel** [comment]
Hey there, I assume you're on a TH Origin and came here because https://github.com/BernhardSchlegel/Brick-32/discussions was not existing until a second ago?
